data/json/npcs/random_encounters/refugee_caravans.json

```diff
@@ -0,0 +1,296 @@
+  },
+  {
+    "type": "npc",
+    "id": "NC_caravan_refugee_C",
+    "//": "Random NPC, demands heroin and can accept.",
+    "name_suffix": "refugee",
+    "class": "NC_SCAVENGER",
+    "attitude": 0,
+    "mission": 7,
+    "chat": "TALK_CARAVAN_REFUGEE_C1",
+    "faction": "wasteland_scavengers"
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_CARAVAN_REFUGEE_A1",
+    "dynamic_line": "Please don't hurt us, we're just trying to get to safety.",
+    "speaker_effect": { "effect": { "npc_first_topic": "TALK_CARAVAN_REFUGEE_A2" } },
+    "responses": [
+      {
+        "text": "What if I want to hurt you?  I'm so good at it and I've had so much practice.",
+        "topic": "TALK_CARAVAN_REFUGEE_A1_Hurt"
+      },
+      { "text": "What's your situation here?", "topic": "TALK_CARAVAN_REFUGEE_A1_Job" },
+      { "text": "I don't have time for zombie bait like you.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_CARAVAN_REFUGEE_A2",
+    "dynamic_line": "You again.  Any good news?  Haven't heard any in a long time.",
+    "responses": [
+      { "text": "What's your story?", "topic": "TALK_CARAVAN_REFUGEE_A_Story" },
+      { "text": "No, I don’t have any.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_CARAVAN_REFUGEE_A1_Hurt",
+    "dynamic_line": "I can’t tell if you’re joking or not, please don’t say such things.",
+    "responses": [
+      { "text": "Oh, I wasn’t joking <name_b>.", "effect": "hostile", "topic": "TALK_DONE" },
+      { "text": "What's your situation here?", "topic": "TALK_CARAVAN_REFUGEE_A1_Job" },
+      { "text": "I’m sorry, I should be going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_CARAVAN_REFUGEE_A1_Job",
+    "dynamic_line": "We're just trying to find a safe place right now.  I think there's a FEMA camp nearby that should have room for us.",
+    "responses": [
+      { "text": "That's it?  What's your story?", "topic": "TALK_CARAVAN_REFUGEE_A_Story" },
+      { "text": "Good luck with that.  I'll be on my way.", "topic": "TALK_DONE" },
+      {
+        "text": "You're never going to make it, you should just end it now.",
+        "topic": "TALK_DONE",
+        "opinion": { "value": -1, "anger": 1 }
+      }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_CARAVAN_REFUGEE_A_Story",
+    "dynamic_line": "We were at the same little evacuation hut, but it was all messed up, just used condoms and broken bottles and lipstick writing on the walls.  We found another group of refugees like us on the road, but they had all been killed, all ripped up, blood everywhere.  We took their food and made it all the way to the refugee center, but they told us there was no room and we should just go back where we came from and fight the dead.  We're not doing that, so we're going to a FEMA camp.  They should have more room and supplies.  No guarantees, but it's a chance.",
```

Comment:
"_the_ refugee center" is the way players talk about this, not people in the game.

The refugee center does not tell people to go and fight zombies.

Reply:
Changed to "a" or  "any"